k-distribution/pl-tutorial/1_k/4_imp++/lesson_8/imp.md

```diff
@@ -563,107 +563,3 @@ This rule acts like a ``tail recursion'' optimization, but for blocks. */
               | "c"     [token]
 endmodule
 ```
-### On Kompilation Options
-
-We are done with the IMP++ semantics. The next step is to kompile the
-definition using the `kompile` tool, this way generating a language
-model. Depending upon for what you want to use the generated language model,
-you may need to kompile the definition using various options. We here discuss
-these options.
-
-To tell the **K** tool to exhaustively explore all the behaviors due to the
-non-determinism of addition, division, and threads, we have to kompile
-with the command:
-
-```shell script
-kompile imp.k --transition="addition division lookup assignment increment read print"
```

Comment:
These tags are unneeded now, so we should remove the corresponding `group(_)` attributes on the rules (and similarly in all the other lessons).